es6/README.md

```diff
@@ -48,9 +80,200 @@ const MaxAllowed = 7;
 const maxAllowed = 7;
 ```
 
+Factor out constants into a constants file if a given [module](#modules) has more than 3 constants:
+
+```js
+// good (imports constants from constants file)
+import * as constants from './constants';
+
+// bad (has more than 3 constants w/in module)
+const FIRST_CONSTANT = 'foo';
+const SECOND_CONSTANT = 'bar';
+const THIRD_CONSTANT = 'baz';
+const FOURTH_CONSTANT = 'qux';
+const FIFTH_CONSTANT = 'quux';
+const SIXTH_CONSTANT = 'corge';
+```
+
+However, if you are using 3 or fewer constants from a file, you can import them individually:
+
+```js
+import {FIRST_CONSTANT, FIFTH_CONSTANT} from './constants';
+```
+
+Avoid using `const` for local variables (eslint: [`prefer-const`](http://eslint.org/docs/rules/prefer-const)):
+
+```js
+// good
+const generateGreeting = (name=DEFAULT_NAME) => {
+    let formattedNow = (new Date()).toString();
+
+    return `Hi, ${name} on ${formattedNow}`;
+}
+
+// bad (uses `const` for `formattedNow` local variable)
+const generateGreeting = (name=DEFAULT_NAME) => {
+    const formattedNow = (new Date()).toString();
+
+    return `Hi, ${name} on ${formattedNow}`;
+}
+```
+
+There is a pattern in the industry to use `const` if a variable is never going to be reassigned within a block (see eslint [`prefer-const`](http://eslint.org/docs/rules/prefer-const)). However, this is an abuse of `const` as it is intended for variables that are truly constant. The motivation for this practice is that it can help enforce "immutability" since a `const` variable cannot be reassigned. But immutability and `const` bindings are two separate things. For instance, an object declared `const` can still have its properties mutated.
+
+For more on constants, read [_Learning ES6: Block-level scoping with `let` and `const`_](http://www.eventbrite.com/engineering/learning-es6-block-level-scoping-with-let-and-const/).
+
+**[⬆ back to top](#table-of-contents)**
+
+## Strings
+
+### Template literals
+
+When building up a string, use a template literal instead of string concatenation (eslint: [`prefer-template`](http://eslint.org/docs/rules/prefer-template.html)):
+
+```js
+// good
+const generateGreeting = (name=DEFAULT_NAME) => {
+    const formattedNow = (new Date()).toString();
```

Comment:
You don't need the .toString() anymore here since you're doing the conversion below, right?